Search/Metadata/ClassMetadata.php

```diff
@@ -78,18 +78,41 @@ public function getIndexMetadata($contextName)
         return $this->indexMetadatas[$contextName];
     }
 
-    public function serialize()
+    public function __serialize(): array
+    {
+        $data = [
```

Comment:
calling `parent::__serialize` does end in a segmentation fault so I reimplemented the logic here.